src/main.rs

```diff
@@ -1,13 +1,17 @@
-use std::path::PathBuf;
+use std::{path::PathBuf, process::exit};
 
 use jakt::{Compiler, JaktError, Span};
 
+use pico_args::Arguments;
```

Comment:
Isn't the usual `use` order standard library, third-party, first-party?

Reply:
idk, rustfmt doesn't complain 🤷

Reply:
Might not be the default, but I think there's a config option for it, I remember using it for serenity-rust's config.